sdk/tables/azure-data-tables/azure/data/tables/_table_client.py

```diff
@@ -210,13 +213,13 @@ def set_table_access_policy(
         signed_identifiers = identifiers  # type: ignore
         try:
             self._client.table.set_access_policy(
-                table=self.table_name, table_acl=signed_identifiers or None, **kwargs
+                table=self.table_name, table_acl=signed_identifiers or None, **kwargs  # type: ignore
             )
         except HttpResponseError as error:
             try:
                 _process_table_error(error)
             except HttpResponseError as table_error:
-                if (table_error.error_code == 'InvalidXmlDocument'
+                if (table_error.error_code == 'InvalidXmlDocument'  # type: ignore
```

Comment:
Does HttpResponseError have an error_code on it? I think it was accessed as table_error.error.code

Reply:
Tables has some custom error processing that adds this property.